README.md

```diff
@@ -32,6 +32,17 @@ Se debe ejecutar el comando
 make check
 ```
 
+## Test-runner
+Se ha elegido `Cypress`, la toma de decisión documentada se puede ver en [este fichero](./docs/test_runners.md).
```

Comment:
cypress es para testing end to end. No tiene ningún sentido usarlo para tests unitarios. cypress crea una página e incluye el código en la misma. Aparte de que es un test muy pesado, no aporta nada desde el punto de vista de tests unitarios, porque usa Chai como prácticamente todos.

Esto es consecuencia de no seguir el procedimiento explicado repetidamente en clase: enunciar los criterios y requisitos, probar las alternativas, y evaluar objetivamente los resultados.

Reply:
De acuerdo, quizá me dejé llevar en la decisión al ver en muchos foros que esta era la alternativa que más está creciendo y que muchos desarrolladores están migrando a ella. Lo cambiaré definitivamente.

El hecho de que pudiese hacer test unitarios no equivale a que sea adecuado en este caso. Muchas gracias por sus comentarios :smiley: